src/cmd/script_args/type_args.rs

```diff
@@ -0,0 +1,174 @@
+use clap::Parser;
+use move_core_types::language_storage::TypeTag;
+use move_core_types::parser::parse_type_tag;
+use std::{fmt, str::FromStr};
+
+/// TypeTag vector container.
+#[derive(Debug, Parser)]
+pub struct TypeArgVec {
+    /// TypeTag arguments separated by spaces.
+    ///
+    /// Example: `u8 u16 u32 u64 u128 u256 bool address vector signer`.
+    #[clap(long, multiple_values(true))]
+    pub(crate) type_args: Vec<MoveType>,
+}
+
+// This function cannot handle the full range of types that MoveType can
```

Comment:
Not about the comment, but when reading this, I just wondered, if we have the same editor width (allowed number of characters in one line) in all of our projects :thinking:

Reply:
We don't have that yet. We could, but I guess we don't care yet - since it's only a few of us working here - so keeping the code style consistency is easy. But a good point